tests/kola/systemd/test.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+## kola:
+##   exclusive: false
+#
+# Checks if default shutdown time is 1m 30s
+#
+# See: https://github.com/coreos/fedora-coreos-tracker/issues/1404
+
+set -xeuo pipefail
+
+. $KOLA_EXT_DATA/commonlib.sh
+
+DefaultTimeoutStartUSec=$(systemctl show --value --property=DefaultTimeoutStartUSec) #$timeout
+DefaultTimeoutStopUSec=$(systemctl show --value --property=DefaultTimeoutStopUSec)
+DefaultTimeoutAbortUSec=$(systemctl show --value --property=DefaultTimeoutAbortUSec)
+DefaultDeviceTimeoutUSec=$(systemctl show --value --property=DefaultDeviceTimeoutUSec) 
+
+error="DefaultTimeoutStartUsec=$DefaultTimeoutStartUSec\nDefaultTimeoutStopUSec=$DefaultTimeoutStopUSec\nDefaultTimeoutAbortUSec=$DefaultTimeoutAbortUSec\nDefaultDeviceTimeoutUSec=$DefaultDeviceTimeoutUSec"
+
+timeout="1min 30s"
+
+# Verify that the timeouts are 1 minute and 90 seconds (150 seconds)
+
+if [[ "$DefaultTimeoutStartUSec" == "$timeout" ]]; then
+  ok "The default start timeout is 1m 30 seconds."
+else
+  fatal -e $error
```

Comment:
```suggestion
  fatal $error
```

This relies on the implementation of the fatal function. Let's just use a space instead.